1568388875923	Sync.LogManager	DEBUG	Flushing file log
1568388875924	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1568388875927	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1568388875949	Sync.LogManager	DEBUG	Log cleanup threshold time: 1567524875949
1568388875982	Sync.LogManager	DEBUG	Done deleting files.
1568388876885	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1568388877948	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1568388905313	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1568388905313	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1568388905321	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1568388906314	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1568388911380	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1568388911381	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1568388911389	Sync.Service	DEBUG	User-Agent: Firefox/69.0 (Linux x86_64) FxSync/1.71.0.20190827005903.desktop
1568388911389	Sync.Service	INFO	Starting sync at 2019-09-13 21:35:11 in browser session 7D8v64NbtuG7
1568388911389	Sync.Service	DEBUG	In sync: should login.
1568388911389	Sync.Service	INFO	User logged in successfully - verifying login.
1568388911390	Sync.BrowserIDManager	DEBUG	unlockAndVerifyAuthState already has (or can fetch) sync keys
1568388911390	Sync.Status	DEBUG	Status.login: error.login.reason.network => success.status_ok
1568388911390	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1568388911390	Sync.Service	DEBUG	Fetching unlocked auth state returned success.status_ok
1568388911390	Sync.BrowserIDManager	INFO	Getting keys
1568388911391	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1568388911391	FirefoxAccounts	DEBUG	enter getAssertion()
1568388911396	FirefoxAccounts	DEBUG	got keyPair
1568388911397	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1568388911399	Services.Common.RESTRequest	TRACE	local time, offset: 1568388911399, 0
1568388911402	Services.Common.RESTRequest	DEBUG	POST request to https://api.accounts.firefox.com/v1/certificate/sign
1568388911402	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1568388911402	Services.Common.RESTRequest	TRACE	HTTP Header content-type: application/json
1568388911402	Services.Common.RESTRequest	TRACE	HTTP Header accept-language: en-US, en
1568388911402	Services.Common.RESTRequest	DEBUG	POST Length: 888
1568388911402	Services.Common.RESTRequest	TRACE	POST Body: {"publicKey":{"algorithm":"DS","y":"B8C83033190258C99DEC9CF087D00C1762827B7CB1BB31928F268BEEFAF8E92BBFC9A727417D81BA5942151000694FC5B44B09BF47626A1A09C93E46DD2A716822E1B90CC2C58FE4EFF3705173161F3957826ADB3811661C6684F89AD1039838552A8BBBC7CBCD539C1214396B223FCAD462B1A11819935089595B705E045D9B","p":"FF600483DB6ABFC5B45EAB78594B3533D550D9F1BF2A992A7A8DAA6DC34F8045AD4E6E0C429D334EEEAAEFD7E23D4810BE00E4CC1492CBA325BA81FF2D5A5B305A8D17EB3BF4A06A349D392E00D329744A5179380344E82A18C47933438F891E22AEEF812D69C8F75E326CB70EA000C3F776DFDBD604638C2EF717FC26D02E17","q":"E21E04F911D1ED7991008ECAAB3BF775984309C3","g":"C52A4A0FF3B7E61FDF1867CE84138369A6154F4AFA92966E3C827E25CFA6CF508B90E5DE419E1337E07A2E9E2A3CD5DEA704D175F8EBF6AF397D69E110B96AFB17C7A03259329E4829B0D03BBC7896B15B4ADE53E130858CC34D96269AA89041F409136C7242A38895C9D5BCCAD4F389AF1D7A4BD1398BD072DFFA896233397A"},"duration":21600000}
1568388911405	Services.Common.RESTRequest	TRACE	onStartRequest: POST https://api.accounts.firefox.com/v1/certificate/sign
1568388911405	Services.Common.RESTRequest	TRACE	Channel for POST https://api.accounts.firefox.com/v1/certificate/sign returned status code 2152398878
1568388911405	Services.Common.RESTRequest	DEBUG	POST https://api.accounts.firefox.com/v1/certificate/sign failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1568388911405	Services.Common.RESTRequest	TRACE	POST body: 
1568388911405	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:487
1568388911406	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 623"  data: no] Stack trace: get status()@resource://services-common/rest.js:623
request()@resource://services-common/hawkclient.js:239
1568388911406	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 638"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:638
request()@resource://services-common/hawkclient.js:247
1568388911408	Hawk	DEBUG	(Response) /certificate/sign: code: null - Status text: null
1568388911409	Services.Common.RESTResponse	TRACE	Processing response headers.
1568388911409	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 670"  data: no] Stack trace: get headers()@resource://services-common/rest.js:670
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:255
1568388911409	Services.Common.RESTResponse	TRACE	Processing response headers.
1568388911409	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 670"  data: no] Stack trace: get headers()@resource://services-common/rest.js:670
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:256
1568388911409	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 638"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:638
_constructError()@resource://services-common/hawkclient.js:122
request()@resource://services-common/hawkclient.js:261
1568388911410	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 623"  data: no] Stack trace: get status()@resource://services-common/rest.js:623
_constructError()@resource://services-common/hawkclient.js:123
request()@resource://services-common/hawkclient.js:261
1568388911410	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 623"  data: no] Stack trace: get status()@resource://services-common/rest.js:623
_constructError()@resource://services-common/hawkclient.js:124
request()@resource://services-common/hawkclient.js:261
1568388911410	Services.Common.RESTResponse	TRACE	Processing response headers.
1568388911410	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 670"  data: no] Stack trace: get headers()@resource://services-common/rest.js:670
_constructError()@resource://services-common/hawkclient.js:130
request()@resource://services-common/hawkclient.js:261
1568388911410	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568388911411	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568388911411	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568388911411	Sync.Status	DEBUG	Status.login: success.status_ok => error.login.reason.network
1568388911411	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1568388911412	Sync.BrowserIDManager	INFO	Failed to fetch the cluster URL: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568388911412	Sync.Service	DEBUG	verifyLogin failed: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568388911412	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1568388911412	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1568388911412	Sync.ErrorHandler	ERROR	Sync encountered a login error
1568388911413	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1568388911415	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1568388911417	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.network(resource://services-sync/service.js:984:15) JS Stack trace: onNotify@service.js:984:15
1568388911418	Sync.Service	DEBUG	Not syncing: login returned false.
